lib/interface/helpers.rb

```diff
@@ -2,17 +2,47 @@
 
 module Interface
   module Helpers
-    # Errors raised here identify the class_name that is not implementing
-    # the method required by the Interface.
+    # Errors raised here identify the class_name that is not implementing or is
+    # implemented with the wrong arity required by the Interface.
     def must_implement(*args)
-      args.each do |method_name|
-        next if respond_to? method_name
-        raise Interface::Error::NotImplementedError.new(
-                class_name: self.class.name,
-                method_name: method_name,
-                interface_name: self.class.ancestors.first
-              )
+      args.each do |method|
+        if method.is_a? Hash
```

Comment:
It doesn't work. It happens what I pointed in this [comment](https://github.com/bluegod/rint/pull/1#discussion_r77570534).

For example, let we `p "#{method_name} - #{arity}"` in this code, getting the args from `must_implement :play, play_quietly: 1, another_play: 0`. The output it'll be:
```text
"play - "
"{:play_quietly=>1, :another_play=>0} - " 
```